jobs/ci-kubernetes-e2e-gce-federation-serial.env

```diff
@@ -8,7 +7,6 @@ KUBE_NODE_OS_DISTRIBUTION=gci
 
 # Where the clusters will be created.
 E2E_ZONES=us-central1-a us-central1-b us-central1-f
-FEDERATION_CLUSTERS=us-central1-a us-central1-b us-central1-f
```

Comment:
My goal is to replace `E2E_ZONES` with `FEDERATION_CLUSTERS`. The trouble with the current approach is that the cluster's name is dependent on the zone in which it runs. This means we cannot have 2 clusters in the same zone in our tests, which is pretty bad. So I want to move to `FEDERATION_CLUSTERS` and in the transition phase we need both.